### PR TITLE
bugfix: baseline is no longer required in kakunin.conf.js

### PR DESCRIPTION
**Pull request by DawidRosol**

### CODE REVIEW

File: src/core/config-schema.helper.ts [hunk 1]
```diff
@@ -41,7 +41,7 @@ const schema = Joi.object({
   regexes: Joi.array().items(Joi.string()),
   hooks: Joi.array().items(Joi.string()),
   imageComparator: {
-    baselineFolder: Joi.string().required(),
+    baselineFolder: Joi.string(),
```
Comment (tgorskievo) [inline src/core/config-schema.helper.ts]: Adam's right. Let's change it to 
``` imageComparator: Joi.object().keys({	
    baselineFolder: Joi.string().required(),	  
    temporaryFolder: Joi.string(),	   
    saveAboveTolerance: Joi.number()	
      .min(0)	   
      .max(99),	   
  }),```

File: MIGRATION-4.0.0.MD [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+example for kakunin.conf.js
+
+```javascript
+  imageComparator: {
+    baselineFolder: 'baseline'
+  },
+  ```
+ 
+ Finally kakunin.conf.js should look like that: [finalnie bedzie wygladac tak:]
```
Comment (tgorskievo) [inline MIGRATION-4.0.0.MD]: `finalnie bedzie wygladac tak:` :D

File: MIGRATION-4.0.0.MD [hunk 3]
```diff
@@ -0,0 +1,83 @@
+#in progress
```
Comment (simonsipl) [inline MIGRATION-4.0.0.MD]: in progress ?

Comment (tgorskievo) [reply MIGRATION-4.0.0.MD]: In the future

Comment (tgorskievo) [reply MIGRATION-4.0.0.MD]: @DawidRosol

File: MIGRATION-4.0.0.MD [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+# Kakunin - automated testing framework
+#### Migration to 4.0.0 version
+
+What needs to be done:
```
Comment (simonsipl) [inline MIGRATION-4.0.0.MD]: It's a confusion for me. I think it should be like:
` To do the migration:
 1. Add for kakunin.conf.js info that imageComparator.baseline is required 
`

File: MIGRATION-4.0.0.MD [hunk 5]
```diff
@@ -0,0 +1,81 @@
+# Kakunin - automated testing framework
+#### Migration to 4.0.0 version
+
+To do the migration:
+1. Add for kakunin.conf.js info that imageComparator.baseline is required 
+
+example for kakunin.conf.js
+
+```javascript
```
Comment (tgorskievo) [inline MIGRATION-4.0.0.MD]: Property that needs to be added to `kakunin.conf.js`

File: MIGRATION-4.0.0.MD [hunk 6]
```diff
@@ -0,0 +1,81 @@
+# Kakunin - automated testing framework
+#### Migration to 4.0.0 version
+
+To do the migration:
+1. Add for kakunin.conf.js info that imageComparator.baseline is required
```
Comment (tgorskievo) [inline MIGRATION-4.0.0.MD]: **imageComparator.baseline** property must be added to kakunin.conf.js

**Pull request merged**